study.md

```diff
@@ -25,7 +25,7 @@ In your own words, define what a SPA is. In your answer, be sure to cite any
 relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+Based on the first reading, a SPA is an app that updates the app's page based on user interactions with the page without having to reload the page.
```

Comment:
Yep, and a key element of this is making only one request to the server, and then dealing with all the data and managing view states on the front end.